engine/modelc/src/java/com/dynamo/bob/pipeline/ModelImporterJni.java

```diff
@@ -444,6 +444,9 @@ public static class FileDataResolver implements DataResolver
         }
 
         public byte[] getData(String path, String uri) {
+            if (uri == null)
+                return null; // Usually compressed buffers, which we don't support yet
```

Comment:
Buffers compressed with Draco or EXT_meshopt buffers require special unpacking.
We'll add support later, but for now we focus on handling the cases gracefully.